content/04-guides/01-database-workflows/08-sql-views.mdx

```diff
@@ -0,0 +1,378 @@
+
+1. Create a new file named `index.js` and add the following code to it:
+
+   ```ts
+   const { PrismaClient } = require('@prisma/client')
+
+   const prisma = new PrismaClient({})
+
+   async function main() {
+     const sarahPosts = await prisma.user.create({
+       data: {
+         name: 'Sarah',
+         email: 'sarah@prisma.io',
+         Post: {
+           create: [
+             { title: 'My first post', published: false },
+             { title: 'All about databases', published: true },
+             { title: 'Prisma Day 2020', published: false },
+           ],
+         },
+       },
+     })
+
+     const emilyPosts = await prisma.user.create({
+       data: {
+         name: 'Emily',
+         email: 'emily@prisma.io',
+         Post: {
+           create: [
+             { title: 'My first post', published: false },
+             { title: 'All about databases', published: true },
+             { title: 'Prisma Day 2020', published: false },
+           ],
+         },
+       },
+     })
+
+     const drafts = await prisma.drafts.findMany({})
+
+     console.log(drafts)
+
+     const filteredDrafts = await prisma.drafts.findMany({
+       where: {
+         email: 'sarah@prisma.io',
+       },
+     })
+
+     console.log(filteredDrafts)
+   }
+
+   main()
+     .catch(e => {
+       throw e
+     })
+     .finally(async () => {
+       await prisma.disconnect()
+     })
+   ```
+
+   This example:
+
+   - Creates two `User` records with three `Post` records each
+   - Returns all `Draft` records from the view
+   - Returns all `Draft` records from the view where the author's email is `emily@prisma.io`
+
+2. Run the code with the following command:
+
+   ```
+   node index.js
+   ```
+
+   The following output indicates that the view works as expected - the first query returns all drafts (`Post` records where `published` is `false`), and the second query returns drafts by `emily@prisma.io` only):
+
+   ```json
+    ALL DRAFTS
+    [
+        {
+            title: 'My first post',
+            id: 37,
+            email: 'sarah@prisma.io',
+            published: false
+        },
+        {
+            title: 'Prisma Day 2020',
+            id: 39,
+            email: 'sarah@prisma.io',
+            published: false
+        },
+        {
+            title: 'My first post',
+            id: 40,
+            email: 'emily@prisma.io',
+            published: false
+        },
+        {
+            title: 'Prisma Day 2020',
+            id: 42,
+            email: 'emily@prisma.io',
+            published: false
+        }
+    ]
+    FILTERED DRAFTS
+    [
+        {
+            title: 'My first post',
+            id: 37,
+            email: 'sarah@prisma.io',
+            published: false
+        },
+        {
+            title: 'Prisma Day 2020',
+            id: 39,
+            email: 'sarah@prisma.io',
+            published: false
+        }
+    ]
```

Comment:
`ALL DRAFTS` and `FILTERED DRAFTS` breaks the highlighting here I think - also does not really come from the code. Maybe as comments for clarity?